Plugins/GRpluginStands.txt

```diff
@@ -1739,178 +2020,339 @@ PRIORITY:+1
+COORD:N063.27.14.301:E010.54.43.194
+COORD:N063.27.15.503:E010.54.43.178
 ENGINETYPE:PT
 CODE:C
 PRIORITY:+2
 ADEP:EN
 
 //STAND 28
-STAND:ENVA:28:N063.27.16.370:E010.54.45.920:18
+STAND:ENVA:28
+COORD:N063.27.17.038:E010.54.43.187
+COORD:N063.27.17.020:E010.54.46.774
+COORD:N063.27.15.676:E010.54.46.767
+COORD:N063.27.15.689:E010.54.43.209
+COORD:N063.27.17.038:E010.54.43.187
 ENGINETYPE:PT
 CODE:C
 PRIORITY:+2
 ADEP:EN
 
 //STAND 29
-STAND:ENVA:29:N063.27.17.790:E010.54.45.980:18
+STAND:ENVA:29
+COORD:N063.27.17.197:E010.54.43.192
+COORD:N063.27.18.647:E010.54.43.355
+COORD:N063.27.18.618:E010.54.46.788
+COORD:N063.27.17.186:E010.54.46.793
+COORD:N063.27.17.197:E010.54.43.192
 ENGINETYPE:PT
 CODE:C
 PRIORITY:+2
 ADEP:EN
 
 //STAND 30
-STAND:ENVA:30:N063.27.17.600:E010.54.49.140:18
+STAND:ENVA:30
+COORD:N063.27.19.167:E010.54.51.194
+COORD:N063.27.16.843:E010.54.51.197
+COORD:N063.27.16.855:E010.54.48.107
+COORD:N063.27.19.158:E010.54.48.077
+COORD:N063.27.19.167:E010.54.51.194
 PRIORITY:-1
 CODE:D
 ADEP:EN
 	
 //STAND 31
-STAND:ENVA:31:N063.27.17.590:E010.54.53.320:18
+STAND:ENVA:31
+COORD:N063.27.16.837:E010.54.54.966
+COORD:N063.27.16.873:E010.54.52.016
+COORD:N063.27.19.159:E010.54.52.031
+COORD:N063.27.19.149:E010.54.55.006
+COORD:N063.27.16.837:E010.54.54.966
 PRIORITY:-1
 CODE:D
 ADEP:EN
 
 //STAND 32
-STAND:ENVA:32:N063.27.17.580:E010.54.56.820:18
+STAND:ENVA:32
+COORD:N063.27.16.860:E010.54.55.333
+COORD:N063.27.16.812:E010.54.58.114
+COORD:N063.27.19.163:E010.54.58.177
+COORD:N063.27.19.146:E010.54.55.362
+COORD:N063.27.16.860:E010.54.55.333
 CODE:C
 ADEP:EN
 
 //STAND 33
-STAND:ENVA:33:N063.27.17.580:E010.54.59.750:18
+STAND:ENVA:33
+COORD:N063.27.16.835:E010.54.58.480
+COORD:N063.27.16.793:E010.55.01.019
+COORD:N063.27.19.150:E010.55.01.066
+COORD:N063.27.19.153:E010.54.58.477
+COORD:N063.27.16.835:E010.54.58.480
 CODE:C
 ADEP:EN
 
 //STAND 34
-STAND:ENVA:34:N063.27.17.580:E010.55.02.680:18
+STAND:ENVA:34
+COORD:N063.27.16.823:E010.55.01.370
+COORD:N063.27.16.801:E010.55.03.977
+COORD:N063.27.19.158:E010.55.04.010
+COORD:N063.27.19.154:E010.55.01.422
+COORD:N063.27.16.823:E010.55.01.370
 CODE:C
 ADEP:EN
 
 //STAND 35
-STAND:ENVA:35:N063.27.17.580:E010.55.05.600:18
+STAND:ENVA:35
+COORD:N063.27.16.823:E010.55.04.286
+COORD:N063.27.16.795:E010.55.06.865
+COORD:N063.27.19.135:E010.55.06.987
+COORD:N063.27.19.141:E010.55.04.311
+COORD:N063.27.16.823:E010.55.04.286
 CODE:C
 ADEP:EN
 
 //STAND 36
-STAND:ENVA:36:N063.27.17.570:E010.55.08.500:18
+STAND:ENVA:36
+COORD:N063.27.16.817:E010.55.07.204
+COORD:N063.27.16.776:E010.55.09.984
+COORD:N063.27.19.133:E010.55.10.046
+COORD:N063.27.19.131:E010.55.07.315
+COORD:N063.27.16.817:E010.55.07.204
 CODE:C
 ADEP:EN
 
 //STAND 37
-STAND:ENVA:37:N063.27.17.570:E010.55.11.800:18
+STAND:ENVA:37
+COORD:N063.27.16.798:E010.55.10.308
+COORD:N063.27.16.766:E010.55.13.428
+COORD:N063.27.19.119:E010.55.13.562
+COORD:N063.27.19.124:E010.55.10.403
+COORD:N063.27.16.798:E010.55.10.308
 CODE:C
 ADEP:EN
```

Comment:
Are these stands domestic only? They dont have a connection to the terminal?

Reply:
Need to check that. Might be flex stands, but i have never seen them actually used.

Reply:
You are referring to stand 38 and 39, right?

Reply:
Correct